src/configuration/services/network-storage.md

```diff
@@ -0,0 +1,187 @@
+# Network Storage
+
+Platform.sh supports internal "storage as a service" to provide a file store that can be shared between different application containers.
+
+The network storage service enables a new kind of `mount` that refers to a shared service rather than to a local directory.  Any application can use both `local` and/or `service` mounts, or neither.
+
+## Supported versions
+
+* 1.0
+
+(This is a reference to a version of our network storage implementation, not to a version of a 3rd party application.)
+
+## Define the service
+
+First, declare a new service in the `services.yaml` file like so:
+
+```yaml
+files:
+    type: network-storage:1.0
+    disk: 2048
+```
+
+This example creates a service named `files` that is of type `network-storage`, and gives it 2048 MB of storage total.
+
+## Declare the mount
+
+Second, add the following entry to your mounts list:
+
+```yaml
+mounts:
+    'my/files':
+        source: service
+        service: files
+        source_path: network
+```
+
+This block will declare a writeable mount on the application container at the path `my/files`, which will be provided by the `files` service defined above.  The `source_path` specifies the path within the network service that the mount points to.  It is often easiest to have it match the name of the mount point itself but that is not required.
+
+Note that you do *not* need to add a relationship to point to the `files` service.  That is handled automatically by the system.
```

Comment:
Maybe "That is handled automatically by the system" should be "That is made implicit by the mount declaration" or something similar?

Reply:
Is there a relationship added in $PLATFORM_RELATIONSHIPS?  If so, then yes it should be reworded.  If not, then implying that a relationship is added is misleading for the customer as it's not, from their POV.

Reply:
hmmm no there isn't... yeah idk, I guess it's fine as is, that just sort of confused me first time I read it